### PR TITLE
Welcome flow

### PR DESCRIPTION
**Pull request by yhaspel**
initial work for the login flow pages

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@
 .env.test.local
 .env.production.local
 .eslintcache
+package-lock.json
```
Comment (IlyaVi) [inline .gitignore]: remove it from ignore

Comment (yhaspel) [reply .gitignore]: np

File: app/components/CenterCard.js [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+  color: ${smColors.green};
+  cursor: pointer;
+  ${StyledAction}
+`;
+
+class CenterCard extends Component<Props, State> {
+  props: Props;
+
+  state = {
+    password: undefined,
```
Comment (IlyaVi) [inline app/components/CenterCard.js]: pls make assignment to null not undefined - it bad coding practice

File: app/components/CenterCard.js [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+    );
+  }
+
+  renderCardBody = (page: number) => {
+    const { onPress } = this.props;
+    const { password, verifyPassword } = this.state;
+    const canEncrypt = !!password && password === verifyPassword;
+    const hasError = password !== verifyPassword || (!!password && password.length < 8);
+
+    switch (page) {
```
Comment (IlyaVi) [inline app/components/CenterCard.js]: pls separate cases to render functions

**Pull request merged**